barter-data/src/exchange/coinbase/channel.rs

```diff
@@ -17,6 +18,11 @@ impl CoinbaseChannel {
     ///
     /// See docs: <https://docs.cloud.coinbase.com/exchange/docs/websocket-channels#match>
     pub const TRADES: Self = Self("matches");
+    /// [`Coinbase`] real-time L1 orderbook channel.
+    ///
+    /// See docs: <https://docs.cdp.coinbase.com/exchange/docs/websocket-channels/#ticker-channel>
+    pub const ORDER_BOOK_L1: Self = Self("ticker");
+    pub const ORDER_BOOK_L2: Self = Self("level2_batch");
```

Comment:
Nit pick: could you add whitespace between each of the `const`s, and also include a link to the level2_batch.